custom/src/test/java/com/splunk/opentelemetry/SplunkConfigurationTest.java

```diff
@@ -172,12 +173,14 @@ private static void verifyThatMicrometerInstrumentationsAreDisabled(ConfigProper
             "micrometer-splunk",
             "oracle-ucp-splunk",
             "tomcat-jdbc-splunk",
-            "vibut-dbcp-splunk")) {
+            "vibur-dbcp-splunk")) {
       assertInstrumentationDisabled(config, instrumentationName);
     }
   }
 
   private static void assertInstrumentationDisabled(ConfigProperties config, String name) {
-    assertFalse(config.getBoolean("otel.instrumentation." + name + ".enabled", false));
```

Comment:
I noticed that these assertions were pretty crappy, and there were several errors here -- so I replaced them all with assertj; it should be a bit easier to catch these kind of errors now